crates/storage/libmdbx-rs/src/transaction.rs

```diff
@@ -579,19 +579,10 @@ impl TransactionPtr {
         self.timed_out.store(true, std::sync::atomic::Ordering::SeqCst);
     }
 
+    /// Acquires the inner transaction lock to guarantee exclusive access to the transaction
+    /// pointer.
     fn lock(&self) -> MutexGuard<'_, ()> {
-        if let Some(lock) = self.lock.try_lock() {
-            lock
-        } else {
-            tracing::debug!(
```

Comment:
let's instead turn it into a trace log, they can still be useful for figuring out some issues

Reply:
sure, can just move to trace